mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -188,7 +188,9 @@ trait OverrideCompletions { this: MetalsGlobal =>
         }
 
       val history = new ShortenedNames(
-        lookupSymbol = { name => context.lookupSymbol(name, _ => true) :: Nil },
+        lookupSymbol = { name =>
+          context.lookupSymbol(name, sym => !sym.isStale) :: Nil
```

Comment:
Without isStale, we would pick up classes that are no longer here. This actually showed up in `CompletionMatchSuite.stale2`, which seems to now have worked correctly.